openc3-cosmos-init/plugins/packages/openc3-vue-common/src/components/Openc3Screen.vue

```diff
@@ -779,4 +779,8 @@ export default {
 .v-textarea :deep(textarea) {
   padding: 5px;
 }
+.v-textarea .v-field__input {
+  -webkit-mask-image: unset;
+  mask-image: unset;
+}
```

Comment:
What does this change?

Reply:
Removes that little drop shadow on the error screen